recipe/condor_config.generic.conda.patch

```diff
@@ -0,0 +1,59 @@
+--- a/src/condor_examples/condor_config.generic
++++ b/src/condor_examples/condor_config.patched
```

Comment:
Maybe (in the future) worth contributing to https://github.com/htcondor/htcondor/tree/master/src/condor_examples and instead patching from upstream?

Reply:
I can consider that, thanks @jasoncpatton.